examples/python/azure/cdktf.json

```diff
@@ -0,0 +1,6 @@
+{
+  "language": "python",
+  "app": "pipenv run ./main.py",
+  "terraformProviders": ["aws@~> 2.0", "azurerm@~> 2.0.0"],
```

Comment:
The `aws` provider can be dropped from that list.

Reply:
Done.